plugins/site-example/build.gradle

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
Not sure what you mean. That page is already up to date (it does not reference site plugins).